scripts/signer/signer.ps1

```diff
@@ -0,0 +1,87 @@
+#
```

Comment:
This script should be in a PAL windows path since it only applies to windows, e.g. `scripts/signed/Platform/Windows/signer.ps1`

From your comment, instead of making it `python`, I would make it cmake directly. That will make it easier to reuse and to have functions per platform and call that directly.

The only thing that I can see that may be needed is to get the certificate from the local store, seems it can be done with something like `certutil -store My`

Reply:
yeah, can be done in a separate task, I just want to avoid having so many different scripting/bash languages around
yeah, you can call the cmake directly with `cmake -P`